template/chatWindow.html

```diff
@@ -51,7 +51,7 @@
             </div>
          </div>
          <div class="jsxc_transfer jsxc_otr jsxc_disabled" />
-         <input type="text" class="jsxc_textinput" data-i18n="[placeholder]Message" />
+         <textarea class="jsxc_textinput" data-i18n="[placeholder]Message"></textarea>
```

Comment:
Textarea is not allowed to be self closing, because it can contain text. Probably jquery would take care of this, but I am used to it.